apps/platform-app/src/app/[locale]/(profile)/(components)/CookieBanner/CookieBanner.module.scss

```diff
@@ -0,0 +1,68 @@
+.bannerWrapper {
+  position: fixed;
+  bottom: 41px;
+  right: 35px;
+  z-index: 1000;
+  display: flex;
+  width: 500px;
+  // height: 322px;
+  padding: 56px;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 22px;
+  flex-shrink: 0;
+
+  border-radius: 9px;
+  border: 1px solid #d9dbe9;
+  background: rgba(255, 255, 255, 0.96);
+  box-shadow: 0px 5px 14px 0px rgba(8, 15, 52, 0.04);
+
+  @include mobile {
```

Comment:
**suggestion**: Consider using a mixin or variable for the z-index value to maintain consistency across the application.